qchem/pennylane_qchem/_version.py

```diff
@@ -16,4 +16,4 @@
 Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.20.0-dev"
+__version__ = "0.22.0-dev"
```

Comment:
Note: this was mistakenly changed to `"0.20.0-dev"` when merging the release candidate after the `v0.21.0` release. That's the reason why we're jumping from `"0.20.0-dev"` to `"0.22.0-dev"`.